public/js/types.js

```diff
@@ -13,8 +13,8 @@
  */
 export type Source = {
   id: string,
-  url?: string,
-  sourceMapURL?: string
+  url: string,
+  sourceMapURL: string
```

Comment:
nice catch, was thinking it would be an empty string, but if the field is missing it should be `?`